SwiftWisdom/Core/UIKit/NibLoadableView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NibLoadableView.swift
+//  SwiftWisdom
+//
+//  Created by Christian Howland on 3/31/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+/**
+ Subclassing NibLoadableView allows you to automate the loading of a Nib into a placeholder view inside
+ another Nib, using only the Custom Class attribute.
+ 
+ It removes the need to perform the following manual steps every time you want a Nib that can load into a placeholder View:
+
+ - Setting an outlet for the top level view
+ - Setting 'translatesAutoresizingMaskIntoConstraints' to false
+ - Adding constraints to fit the new view into its container
+ - Writing boilerplate Nib loading code.
+
+ The minimum amount of code required to use this would just be the inheritance line, i.e.:
+
+ " YourCustomView: NibLoadableView {} "
```

Comment:
I should add that in general my 2c is that I'd rather not have any additional views lying around whose sole purpose is this kind of containment(for example, top level view in scrollview), but I understand that the manual programmatic autolayout is not everyone's cup of tea, so Im in support of whatever we can do to make lives easier!